BodoSQL/bodosql/bodosql_types/glue_catalog.py

```diff
@@ -119,7 +119,7 @@ def unbox_glue_catalog(typ, val, c):
 @numba.jit
 def get_conn_str(warehouse):
     """Get the connection string for a Glue Iceberg catalog."""
-    return f"glue?warehouse={warehouse}"
+    return f"iceberg+glue?warehouse={warehouse}"
```

Comment:
Basically by this point there is no difference between the runtime and compile time connection string. That was an annoying and unnecessary requirement anyways